artifacts/opendirectory_version.py

```diff
@@ -0,0 +1,30 @@
+import subprocess
+
+factoid = 'opendirectoryd_version'
+
+def fact():
+    '''
+    Returns the "project version" number used to build opendirectoryd
+    per https://support.apple.com/en-gb/HT208315 to check that
+    "Security Update 2017-001" is installed
+    '''
+
+    result = 'None'
+
+    try:
+        proc = subprocess.Popen(
+                ['/usr/bin/what', '/usr/libexec/opendirectoryd'],
+                 stdout=subprocess.PIPE,
+                 stderr=subprocess.PIPE
+                )
+        stdout, _ = proc.communicate()
+    except (IOError, OSError):
+        stdout = None
+
+    if stdout:
+        result = stdout.splitlines()[-1].split(':')[-1]
```

Comment:
To match other "version" facts in this repo I would change this to:
```python
result = stdout.splitlines()[-1].split('-')[-1]
```
```python
Old 
<result>opendirectoryd-483.20.7</result>
vs new
<result>483.20.7</result>
````

Reply:
TIL about /usr/bin/what

As for the PR, I really don't care as long as it doesn't break the run :)

Reply:
I must have skimmed past that. I personally would prefer your recommendation to make rename this artifact. Also didn’t even realize/ think to look at the --version output.